brain/converter/crypto.py

```diff
@@ -40,10 +69,14 @@ def hashpass(p):
     @salt - a random string of saltlength bytes generated to hash the password
 
     '''
-    salt_length = load_salt(app=False)['salt_length']
-    salt_length = salt_length if salt_length is not None else 32
-    salt = base64.b64encode(os.urandom(salt_length))
-    return hashlib.sha512(salt + p).hexdigest()+"$"+salt
+    salt = getsalt(app=False)
+    N,r,p = getscryptparams(app=False)

```

Comment:
This is nice, and cleaner syntax then using `dict` keys.  I think PEP8 will complain about spacing between `,`.